toolkits/springernature/packages/springernature-submission-header/scss/50-components/core.scss

```diff
@@ -0,0 +1,24 @@
+.c-submission-details-header {
+	font-size: 12px;
```

Comment:
do you have font sizes defined in your brand context? as they could be referenced here

Reply:
I did think about that.
There is still some disconnect between the SN brand context and the project CSS. The brand context uses a different base font size, and has a mixture of ems and rems. 
This needs to be pulled together in a later commit.